### ISSUE TITLE
[Bug] Lots of accessibility issues

### ISSUE THREAD

**Issue by marcusosterberg** [piwik/piwik#25049]
### What happened?

<img width="880" height="956" alt="Screenshot from Axe DevTools that concludes a view in Matomo v6 has almost 100 issues" src="https://github.com/user-attachments/assets/d8e2f9ac-1aab-448b-aa60-8c80217acc3d" />

According to automated accessibility tools such as Axe DevTools there are plenty of accessibility issues. In above image I took a look at Matomo v 6, but v 5 is pretty much identical.

This is an obvious problem for organisations that are regulated to be accessible and a showstopper in some procurements.

### What should happen?

No surprising issues should be present. "Surprising" as in not having already had a manual check explaining that it really is not an issue.

### How can this be reproduced?

Go to the default dashboard. Have DevTools installed: https://chromewebstore.google.com/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd

### Matomo version

6.0.0-b1

### Relevant log output

```shell

```

### Validations

- [x] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [x] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [x] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [x] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by des-innocraft** [comment]
Hi @marcusosterberg ,

Thanks for reporting this. Could you please share the Axe DevTools results, including the individual violations and affected elements?

Have you manually verified any of the findings, particularly whether any functionality is difficult or impossible to use with keyboard navigation or a screen reader? If you've tested with a screen reader, could you also provide the screen reader/browser combination?

You mentioned accessibility can be a showstopper for procurement. Are there particular findings here that fail a required accessibility standard (e.g. WCAG 2.1/2.2 AA) or have been identified as procurement blockers?

Finally, do the same violations appear elsewhere in Matomo, or are they limited to the Dashboard?

**Comment by marcusosterberg** [comment]
There are WCAG violations on each and every page I've ever checked, and to many to report them all individually. Can I suggest another approach than listing all of them?

A consistent violation is the contrast ratio on the active element in the menu, widget buttons and the link to the marketplace:
<img width="249" height="351" alt="Image" src="https://github.com/user-attachments/assets/5666bbfa-c7b9-450b-b99c-5701cacf4124" />

<img width="423" height="380" alt="Image" src="https://github.com/user-attachments/assets/73cf2379-6b7f-49b8-b9c3-1e1d81ddbb0b" />
<br>
<img width="139" height="145" alt="Image" src="https://github.com/user-attachments/assets/a7f4b616-d76c-4a1f-a991-34a200ccefb3" />

The datepicker's buttons need discernible text:
<img width="455" height="225" alt="Image" src="https://github.com/user-attachments/assets/9daee242-ff67-4b5a-b279-6fcf8d7e308b" />

Select elements need accessible names:
<img width="318" height="253" alt="Image" src="https://github.com/user-attachments/assets/8f43e45e-63c1-4e32-8362-633e15893345" />
